pex/pep_376.py

```diff
@@ -342,6 +342,10 @@ def reinstall(
         N.B.: A record of reinstalled files is returned in the form of an iterator that must be
         consumed to drive the installation to completion.
 
+        If there is an error re-installing a file due to it already existing in the destination
```

Comment:
N.B.: This was the (undocumented) contract pre-refactor in #1632 and it was kept for link / copy, just not for symlink.